ports/nrf/drivers/bluetooth/download_ble_stack.sh

```diff
@@ -62,16 +56,16 @@ SCRIPT_DIR="$(cd -P "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 if [ $# -eq 0 ]; then 
     echo "No Bluetooth LE stack defined, downloading all."
     download_s110_nrf51_8_0_0 ${SCRIPT_DIR}
-    download_s132_nrf52_6_1_1 ${SCRIPT_DIR}
-    download_s140_nrf52_6_1_1 ${SCRIPT_DIR}
+    download_s132_nrf52_7_2_0 ${SCRIPT_DIR}
+    download_s140_nrf52_7_2_0 ${SCRIPT_DIR}
 else 
     case $1 in
         "s110_nrf51" )
             download_s110_nrf51_8_0_0 ${SCRIPT_DIR} ;;
         "s132_nrf52_6_1_1" )
```

Comment:
You should probably update the name here to `s132_nrf52_7_2_0` as you did for s140 below.

Reply:
Oh!, sorry about that. I missed that one. Strangely It did compile without error.  I will fix it.